### ISSUE TITLE
feat(patch): Add the sorting buttons to all channels 

### ISSUE THREAD

**Issue by jakito1** [ReVanced/revanced-patches-template#636]
### Application

YouTube 

### Issue

Currently most music channels don't allow to sort (by most popular, for example). But you can sort it using the URL. The button just isn't present.

This comment (and the thread) explains better: [Link](https://www.reddit.com/r/youtube/comments/kgxdhf/why_cant_we_sort_by_most_popular_or_oldest_first/gvaj1b4/) 

Channel that supports sorting: [LTT](https://youtube.com/c/LinusTechTips)

Channel that doesn't supports sorting: [2WEI](https://youtube.com/c/2WEIMusic) 

### Patch

Basically, getting the buttons to allow the video sorting on all channels.

### Motivation

As someone who likes to discover new music on YouTube, when I find an artist I like I want to listen to their most popular songs first, but it's rarely possible.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by Moolji** [comment]
so how is this coming along? I see that some other users requested it as well but how is the project to bring back the "sort by oldest" coming along?

**Comment by Poopooracoocoo** [comment]
@Moolji This is a feature request, not a pull request. This is a project run by volunteers. There is no project that you speak of to bring back that specific feature. If someone wants that feature too, they'll try to do it and then they'll make a PR/commit. Otherwise you're out of luck.

**Comment by Moolji** [comment]
> @Moolji This is a feature request, not a pull request. This is a project run by volunteers. There is no project that you speak of to bring back that specific feature. If someone wants that feature too, they'll try to do it and then they'll make a PR/commit. Otherwise you're out of luck. 

Understood, I hope someone works on it since it was useful. I'm still baffled that youtube took this feature away. If there was a competent manager at YouTube they would work on bringing this back and fire whomever's fault it was that caused it to be removed.

**Comment by janek49** [comment]
Any update on this? The sort by oldest is still supported by the API used in YouTube APK Version 17 as I've used it untill today but it stopped playing some videos. Absolutely hate that they "removed" this feature.

**Comment by Freaku17** [comment]
Hi there, can anyone please work on this?

I used an older YouTube apk and the sort by "oldest" still works, hoping someone can make it as a patch here.

Probably just needs 5 lines of code which can be seen through old YT apk?
